### PR TITLE
Reenables secborgs

### PR DESCRIPTION
**Pull request by Wilchenx**
With #40132 coming secborgs are finally balanced within the game.

Also gives AI some ability to prevent human harm.

:cl: Wilchen
balance: reenables secborgs
/:cl:

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
how are you suppose to headmin if you can't even edit configs properly.

**Comment by Wilchenx** [comment]
This is just proof of concept

File: config/game_options.txt [hunk 1]
```diff
@@ -257,9 +257,6 @@ EVENTS_MIN_PLAYERS_MUL 1
 ## Allow the AI job to be picked.
 ALLOW_AI
 
-## Secborg ###
```
Comment (81Denton) [inline config/game_options.txt]: wat

Comment (81Denton) [reply config/game_options.txt]: just remove the `#` before `DISABLE_SECBORG`

Comment (Wilchenx) [reply config/game_options.txt]: configs are serverside anyway, this PR is just for discussion purposes

**Comment by ExcessiveUseOfCobblestone** [comment]
Welcome to Policy Discussion but Github!

**Comment by 81Denton** [comment]
https://tgstation13.org/phpBB/viewforum.php?f=33

**Comment by epochayur** [comment]
inb4 insta-close

**Comment by Cyberboss** [comment]
Policy discussion? In my repo? I think not

**Comment by Wilchenx** [comment]
i was about to close it and fuck off to forums lol

**Pull request closed without merging**